hathor/merged_mining/bitcoin.py

```diff
@@ -282,7 +283,7 @@ class BitcoinTransactionInput(NamedTuple):
     # Transaction version as defined by the sender. Intended for "replacement" of transactions when information is
     # updated before inclusion into a block.
     sequence: int = SEQUENCE_FINAL  # default value disables nLockTime
-    script_witness: List[bytes] = []
+    script_witness: Tuple[bytes, ...] = ()
```

Comment:
I'd say `tuple()` is easier to understand.

Reply:
Oh! I guess it's subjective, I've never seen `tuple()` without arguments. But it's fine either way for me.

Reply:
My best argument pro `()` is that it's the canonical representation (`repr(tuple())` -> `'()'`).